src/lib/containers/table/datasets/DatasetItemsEditor.tsx

```diff
@@ -461,7 +460,6 @@ export function DatasetItemsEditor(props: DatasetItemsEditorProps) {
     {
       key: 'status',
       width: 80,
-      dataKey: 'entityId',
```

Comment:
For the Dataset Editor, the change is simple. By specifying this `dataKey`, we were omitting versionNumber passed to the `BadgeIconsRenderer` component. When we remove `dataKey`, the renderer gets all of the row data that was already present, including the `entityId` and `versionNumber`.